### DISCUSSION TITLE
Allow logs to be streamed for in progress builds

### DISCUSSION THREAD

**Discussion by jpike88** [deszip/BitBot#37] [Feature requests]

**Comment by deszip** [comment]
Hi @jpike88 , actually this feature was present in the early versions  and still there but disabled :( Unfortunately there are two issues with it:
- I cant manage to implement smooth scrolling of 50MB+ text chunks in reasonable time
- Bitrise API doesn't have a pagination enabled for logs so when you open live log for running build (which is the most common use case) you get it from the current point and can't scroll back until build finishes

Thanks for a suggestion, I'll move this to discussions and take a look at the old code, maybe it makes sense to enable it as is.
